procScurves.py

```diff
@@ -0,0 +1,50 @@
+import os
+import threading
+from optparse import OptionParser
+
+parser = OptionParser()
+
+parser.add_option("-r", "--run", type="string", dest="runname", default="2017.03.15-15.03.49.081532",
+                  help="Specify Run to Process", metavar="runname")
```

Comment:
I would not specify a default here, and would instead do a check that the option was passed properly.